### PR TITLE
:sparkles: API review pr

### PR DESCRIPTION
**Pull request by shawn-hurley**

### CODE REVIEW

**Comment by pranavgaikwad** [comment]
@shawn-hurley I am fine with separating the two

File: engine/conditions.go [hunk 1]
```diff
@@ -116,12 +115,17 @@ type CustomVariable struct {
...
 
 type Perform struct {
-	Message *string  `yaml:"message,omitempty"`
+	Message Message  `yaml:",inline"`
 	Tag     []string `yaml:"tag,omitempty"`
 }
 
+type Message struct {
+	Text  *string       `yaml:"message,omitempty"`
+	Links []hubapi.Link `yaml:"links,omitempty" json:"links,omitempty"`
```
Comment (fabianvf) [inline engine/conditions.go:124]: Any reason only links has a json tag?

**Review by fabianvf — APPROVED** [review]
Looks good to me, one question but nice work

File: engine/conditions.go (hunk 1 above)
Comment (shawn-hurley) [reply engine/conditions.go:124]: Nope they should both or neither will update!

Comment (shawn-hurley) [reply engine/conditions.go]: Updated, resolving

**Pull request merged**